### ISSUE TITLE
Some questions about data loading 

### ISSUE THREAD

**Issue by ZHE-SAPI** [Dayan-Guan/DA-VSN#7]
Hi,
This is a very enlightening work!!!  @xing0047 @Dayan-Guan 
I want to ask a question~

When I use./TPS/tps/scripts/train.py to read SynthiaSeq or ViperSeq data, I debug the code and find the following phenomena:

I tried to  **print** some variables of __ getitem__ () ,

When the **shuffle** of source_loader = data.DataLoader() is set to **False**, and the **batch_size**=cfg.TRAIN.BATCH_SIZE_SOURCE is set to **1**,

1. It is found that although the batch_ Size=1, but 4 pictures and the first frame corresponding to them are loaded at one time,
Instead of 1 picture and the previous frame.

2. At the same time, it is found that  4 loaded pictures are disordered, such as 2-1-3-4, rather than 1-2-3-4, it seems to violate the settings of shuffle.

Could you please kindly explain my doubt? Thank you very much!!

_The **print** code are as follows:_

![111](https://user-images.githubusercontent.com/52643313/195514036-d4925084-b47f-4e51-9322-60c4049efdd5.jpg)

_The **print** results are as follows，which the order of each run of print is different:_

---index--- 1
---index--- 0
---index--- 2
img_file tps/data/SynthiaSeq/SEQS-04-DAWN/rgb/000002.png
label_file tps/data/SynthiaSeq/SEQS-04-DAWN/label/000002.png
---index--- 3
img_file tps/data/SynthiaSeq/SEQS-04-DAWN/rgb/000001.png
label_file tps/data/SynthiaSeq/SEQS-04-DAWN/label/000001.png
img_file tps/data/SynthiaSeq/SEQS-04-DAWN/rgb/000003.png
label_file tps/data/SynthiaSeq/SEQS-04-DAWN/label/000003.png
img_file tps/data/SynthiaSeq/SEQS-04-DAWN/rgb/000004.png
label_file tps/data/SynthiaSeq/SEQS-04-DAWN/label/000004.png
image_kf tps/data/SynthiaSeq/SEQS-04-DAWN/rgb/000003.png
image_kf tps/data/SynthiaSeq/SEQS-04-DAWN/rgb/000002.png
image_kf tps/data/SynthiaSeq/SEQS-04-DAWN/rgb/000001.png
image_kf tps/data/SynthiaSeq/SEQS-04-DAWN/rgb/000000.png
label_kf tps/data/SynthiaSeq/SEQS-04-DAWN/label/000003.png
label_kf tps/data/SynthiaSeq/SEQS-04-DAWN/label/000002.png
label_kf tps/data/SynthiaSeq/SEQS-04-DAWN/label/000001.png
label_kf tps/data/SynthiaSeq/SEQS-04-DAWN/label/000000.png
